internal/core/services/operation_manager/operation_cancel_functions.go

```diff
@@ -0,0 +1,59 @@
+package operation_manager
+
+import (
```

Comment:
This entire file was just a extraction refactor, no changes here

Reply:
Nice!!